libs/mgjson/Makefile

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright (C) 2021 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=mgjson
+PKG_RELEASE:=1
+
+PKG-BRANCH=main
+PKG_SOURCE_URL=https://github.com/TheRootED24/mgjson.git
```

Comment:
I checked it, your repository is only 3 weeks old. 
- While looking at it, I found that commit messages are vague or should I rather say that all of them looks the same?
- No one reviewed it.
- There is no documentation except the README, which does not say anything.

Reply:
yes, this is fair. This library was a after the fact addition. I made it after it was brought to my attention that the jsonmg library had far more utility as a standalone library, rather then being strapped to the rather large mongoose library. Being a shared library, not to mention, the library itself is part of the very well documented Mongoose C Networking library -->[https://mongoose.ws/documentation/](url) i guess i didn't figure separate documentation was needed at the time. I do however, see your point, i'll add some more general info to my repo with more explaination regarding the library's